### ISSUE TITLE
Will have milestone support Angular 16?

### ISSUE THREAD

**Issue by MC220918915** [tjoskar/ng-lazyload-image#568]
Will have milestone support Angular 16? Since 2 years not being update the code anymore.

**Comment by tjoskar** [comment]
Yeah, unfortunately, I am not actively developing this library anymore, see: https://github.com/tjoskar/ng-lazyload-image/issues/570

But I would welcome help if you can upgrade it to support newer version of Angular.
